dagmc_stats/DagmcStats.py

```diff
@@ -40,3 +40,23 @@ def __init__(self, filename, populate=False):
         self._vol_data = pd.DataFrame(
             columns=['vol_eh', 'surf_per_vol', 'coarseness'])
         self._vol_data = self._vol_data.set_index('vol_eh')
+
+    def get_native_ranges(self, meshset, entity_types):
```

Comment:
If `entity_types` becomes a class variable.
```suggestion
    def get_native_ranges(self, meshset, entity_types = DagmcStats.entity_types):
```

Reply:
Also change this to called `set_native_ranges()`

Reply:
Final (?) suggestion
```suggestion
    def _set_native_ranges(self):
```